.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Bug Report
+about: Report a bug encountered while using CoreDNS 
+labels: bug
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://kubernetes.io/security/
```

Comment:
Is this the correct security contact info?

Reply:
Done